php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S2699.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Tests should include assertions",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "10min"
+  },
+  "tags": [
+    "tests"
```

Comment:
I add the 'phpunit' tag to the new rules. What do you think about it? Some Java rules have the 'junit' tag. We should try to be consistent.